cypress/integration/account/profile-settings.spec.js

```diff
@@ -0,0 +1,158 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const user = {
+  ids: { user_id: 'profile-settings-test-user' },
+  name: 'Test User',
+  primary_email_address: 'test-user@example.com',
+  password: 'ABCDefg123!',
+  password_confirm: 'ABCDefg123!',
+}
+
+describe('Account App profile settings', () => {
```

Comment:
Please consider adding tests with `disable_upload = true` and `use_gravatar = false`. Just UI assertions should be enough.